intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -48,6 +48,8 @@
 def get_feed(feedname, logger):
     # TODO should this be case insensitive?
     feed_idx = {
+        "Open-DB2": scan_db2,
+        "Accessible-HTTP": scan_http,
```

Comment:
Function names should correspond to the feed names.

Reply:
The function names are the feed names appearing in our daily shadowserver CSV file names. The feed names are representing the feed titles from shadowserver.org. In case of Accessible-HTTP, shouldn't it be an accessible service, instead of open?

Reply:
I was referring to the function names (scan_db2, scan_http) which should correspond to the feed names like with other feeds, so I'd suggest
"Open-DB2-Discovery-Service ": open_db2_discovery_service,
"Accessible-HTTP": accessible_http,

Reply:
I understood, it was just my reason for naming them differently - same as the filenames. Ok, I will change it.